python/lsst/analysis/tools/actions/scalar/scalarActions.py

```diff
@@ -39,6 +39,7 @@
     "MedianHistAction",
     "IqrHistAction",
     "DivideScalar",
+    "LoadScalar",
```

Comment:
In context of the commit I pushed (if there is no issues with it you see in your cross review) I think this should just be deleted.